--- a/gtk/main.c	2023-12-22 16:32:42.043672299 +0300
+++ b/gtk/main.c	2023-12-22 16:41:03.470503199 +0300
@@ -20,6 +20,7 @@
  * DEALINGS IN THE SOFTWARE.
  *****************************************************************************/
 
+#include <libintl.h>
 #include <locale.h>
 #include <signal.h>
 #include <string.h>
@@ -56,6 +57,7 @@
 
 #define MY_CONFIG_NAME "transmission"
 #define MY_READABLE_NAME "transmission-gtk"
+#define _(String) gettext (String)
 
 #define SHOW_LICENSE
 static char const* LICENSE =
